sql/snapshot.sql

```diff
@@ -0,0 +1,100 @@
+-- public.global_snapshot definition
+
+-- Drop table
+
+-- DROP TABLE global_snapshot;
+
+CREATE TABLE global_snapshot (
```

Comment:
Notes: 
- `hash` can be VARCHAR
- `subheight` can be int4
- `last_snapshot_hash` can be VARCHAR
- We need to include fields from the current snapshot schema here as nullable fields:
    - `epochProgress`
    - `metagraphSnapshotCount`
- We should have a unique index/constraint on `hash`
- `last_snapshot_hash` should FK to `hash` on this table
- `created_at` / `updated_at` - need DEFAULT NOW() to set initial value and then I think updated_at needs a trigger to update whenever it’s changed. We should consider timestampz instead of timestamp here but maybe it doesn’t matter.